client/src/pages/registration.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import styled from "@emotion/styled";
+
+import Form from "../components/Form";
+import Input from "../components/Input";
+import Button from "../components/Button";
+import Heading from "../components/Heading";
+
+const ContentWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  width: 95vw;
+  height: 100vh;
+  background: ${props => props.theme.colors.secondary};
+`;
+
+const Container = styled.div``;
+
+function AttendeeRegistration() {
+  const [state, setState] = React.useState({
+    name: "",
+    address: "",
+    location: "",
+    email: "",
+    phone: "",
+    workshop1: "",
+    workshop2: "",
+    workshop3: "",
+    else: ""
+  });
+
+  function onChange(event) {
+    const value = event.target.value;
+    setState({
+      ...state,
+      [event.target.name]: value
+    });
+  }
+
+  async function handleSubmit(event) {
+    event.preventDefault();
+    await fetch("http://localhost:8080/attendees", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json"
+      },
+      body: JSON.stringify({
+        state
+      })
+    });
+  }
+
+  return (
+    <ContentWrapper>
+      <Container>
+        <Heading>Anmeldung</Heading>
+        <Form onSubmit={handleSubmit}>
+          <Input autoFocus name="name" onChange={onChange} placeholder="Name, Vorname"></Input>
```

Comment:
Was mit hier noch auffällt: Aktuell sind deine Input-Felder uncontrolled, wenn du den value={state.name} setzt sind Sie controlled. Ich bin mir allerdings nicht sicher, ob es im dem Fall zwingend notwendig ist.

Reply:
Danke :)
Habe das gerade mal probiert:
```<Input value={attendee.name} name="address" onChange={onChange} placeholder="Adresse" />```

Dann kann man nicht mehr in das Input Feld schreiben.
Aber ich habe den Unterschied zwischen controlled und uncontrolled auch noch nicht richtig gecheckt. Muss ich mir noch mal angucken.

Reply:
Solved:
Depends on my input component.